src/io/flutter/view/FlutterView.java

```diff
@@ -104,8 +103,26 @@ public void initToolWindow(@NotNull ToolWindow toolWindow) {
     toolbarGroup.addSeparator();
     toolbarGroup.add(new OverflowActionsAction(this));
 
-    addInspectorPanel("Widgets", InspectorService.FlutterTreeType.widget, toolWindow, toolbarGroup, true);
-    addInspectorPanel("Render Tree", InspectorService.FlutterTreeType.renderObject, toolWindow, toolbarGroup, false);
+    addInspectorPanel(WIDGET_TREE_LABEL, InspectorService.FlutterTreeType.widget, toolWindow, toolbarGroup, true);
+    addInspectorPanel(RENDER_TREE_LABEL, InspectorService.FlutterTreeType.renderObject, toolWindow, toolbarGroup, false);
+
+    listenForRenderTreeActivations(toolWindow);
+  }
+
+  private static void listenForRenderTreeActivations(@NotNull ToolWindow toolWindow) {
+    final ContentManager contentManager = toolWindow.getContentManager();
+    contentManager.addContentManagerListener(new ContentManagerAdapter(){
+      @Override
+      public void selectionChanged(ContentManagerEvent event) {
+        final ContentManagerEvent.ContentOperation operation = event.getOperation();
+        if (operation == ContentManagerEvent.ContentOperation.add) {
+          final String name = event.getContent().getTabName();
+          if (Objects.equals(name, RENDER_TREE_LABEL)) {
+            FlutterInitializer.getAnalytics().sendEvent("inspector", "renderTreeSelected");
```

Comment:
We're tracking what tree we're in here - or when they switch. Should we send for both switching to the render tree, and switching to the widget one?

We also track what screens people are in, so 'flutter inspector', that _could_ map well here? It may be difficult to determine that the screen should be 'flutter inspector / widgets'.

Reply:
Sure.  Will update! 👍